lib/pages/barcode.dart

```diff
@@ -1,8 +1,64 @@
 import 'package:flutter/material.dart';
+import 'package:barcode_scan/barcode_scan.dart';
+import 'dart:async';
+import 'package:flutter/services.dart';
+import 'package:scibble/theme/scibble_color.dart';
+
+class Barcode extends StatefulWidget {
+  @override
+  _ScannerState createState() => _ScannerState();
+}
+
+class _ScannerState extends State<Barcode> {
+  String result = "Varescanner";
+  Future _scanBarcode() async {
+    try {
+      ScanResult barcodeResult = await BarcodeScanner.scan();
+      setState(() {
```

Comment:
Suggest to create a  `setResult` function so you do not have to write `setState( ... )`  all the time. That or make _scanBarcode() return Future<String> and set state when it is returned.